portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,24 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = "I was born in Sydney, Australia.";
 }
+
+var slideIndex = 1;
+showSlide(slideIndex);
+
+function nextSlide(n) {
+  showSlide(slideIndex+=n);
```

Comment:
I would recommend:
```suggestion
 slideIndex+=n;
 showSlide(slideIndex);
```

This is more readable and leaves less room for question on what value is being passed into the function. For example, this made me pause and think about whether slideIndex would be updated before being passed in.

Reply:
Also you update slideIndex in the showSlide so I believe the simplest is:
```suggestion
  showSlide(slideIndex+n);
```

Reply:
Updated this.